modelexpress_client/python/modelexpress/weight_transfer/engine/lazy.py

```
@@ -0,0 +1,234 @@
+        try:
+            meta_args = tuple(
+                torch.empty(a.shape, dtype=a.dtype, device="meta")
+                if isinstance(a, LazyWeight) else a
+                for a in args
+            )
+            out = func(*meta_args, **kwargs)
+            if isinstance(out, torch.Tensor):
+                new_op = OpSpec(name=func_name, args=args[1:], kwargs=kwargs)
+                return LazyWeight(
+                    lazy._lazy_name, out.shape, out.dtype,
+                    lazy._lazy_chain + (new_op,),
```

Ruff (0.15.20)

[warning] Consider `(*lazy._lazy_chain, new_op)` instead of concatenation

Replace with `(*lazy._lazy_chain, new_op)`

(RUF005)